### ISSUE TITLE
Video restarts the app when tapped

### ISSUE THREAD

**Issue by adjustnox** [amzn/fire-app-builder#46]
We use the latest version from last year. We have an issue on Android devices where the app will restart if the user is playing video, and taps anywhere on the video. Any idea what is causing this?

**Comment by levonlevonian** [comment]
Hi there,

Does that happen in the unmodified (no code changes) project, or do you have any custom code around video UI / playback, etc? Would you be able to capture the logs and provide those to us to investigate? Thanks!

**Comment by adjustnox** [comment]
I can tell you that, in the sim at least, it happens in the unmodified default project. I will try to test on device as well.

**Comment by levonlevonian** [comment]
Yes, please test it on a physical device(s) as well, and let us know what you observe. Thanks!

**Comment by adjustnox** [comment]
I've confirmed this happens on device as well.

**Comment by levonlevonian** [comment]
Thanks for checking. Would you be able to provide the feed / videos that you tested it with, and also capture the logs and provide those to us to investigate? If you'd rather not post that info here, then please login to your Dev Portal account, click on "Contact Us" and send your feed details to us privately through that form. Include the URL to this GitHub issue for context. Thanks!
